components/list/item.js

```diff
@@ -75,7 +75,7 @@ const ListItemComponent = sys(
 
 const Item = ({ href, as, ...rest }) =>
   href ? (
-    <Link href={href} passHref as={as}>
+    <Link href={href} passHref as={as} prefetch={false}>
```

Comment:
Good call to not prefetch too many things. FYI, if you upgrade to next.js 9 (which is a good upgrade), they automatically prefetch all links that are visible unless you explicitly say not to (as here).